packages/icons/src/leg-icon.js

```diff
@@ -21,7 +21,8 @@ const LegIcon = ({ getCompanyIcon, leg, ModeIcon, ...props }) => {
 LegIcon.propTypes = {
   // Optional override function for deriving the company icon for a given leg.
   getCompanyIcon: PropTypes.func,
-  leg: coreUtils.types.legType.isRequired,
+  // TYPESCRIPT TODO: restore
+  // leg: coreUtils.types.legType.isRequired,
```

Comment:
I guess since we're basically disabling the react PropTypes it doesn't matter but surely better to have the correct object subtype where possible! Since it doesn't come from core-utils I think it should be fine right